build.gradle

```diff
@@ -28,7 +28,7 @@ repositories {
 
 group = "com.electriccloud"
 description = "Plugins : EC-Azure"
-version = "1.1.0"
+version = "1.2.0"
 
```

Comment:
We don't need to update the version number. 1.1.0 is still in development, we have not published it yet to the catalog.